components/consumers/pdf/Dockerfile

```diff
@@ -1,11 +1,19 @@
-FROM golang:latest
 
-WORKDIR /playwright
-RUN go mod init github.com/ocurity/pdf-consumer &&\
-    go get -u github.com/playwright-community/playwright-go &&\ 
-    go run github.com/playwright-community/playwright-go/cmd/playwright@latest install --with-deps
+# Stage 2: Build
+FROM golang:latest as builder
 
-ENV PATH="${PATH}:/go/pkg/mod/github.com/playwright-community"
-COPY components/consumers/pdf/pdf /playwright/pdf
-COPY components/consumers/pdf/default.html /playwright/default.html
-ENTRYPOINT ["/playwright/pdf"]
\ No newline at end of file
+# Install playwright cli with right version for later use
+RUN go install github.com/playwright-community/playwright-go/cmd/playwright@latest
+
+# Stage 3: Final
+FROM ubuntu:latest
+COPY --from=builder /go/bin/playwright /
+RUN apt-get update && apt-get install -y ca-certificates tzdata \
+    # Install dependencies and all browsers (or specify one)
+    && /playwright install --with-deps \
```

Comment:
as far as I understand the code, we are only invoking Chromium in headless mode but this will install other browsers too. why don't we specify just chromium?